js/types.ts

```diff
@@ -0,0 +1,102 @@
+// Common types
+export type TitleData = {
+  content: string;
+  priority: number;
+};
+export type LinkData = {
+  name: string;
+  url: string;
+};
+export type LinkImage = {
+  name: string;
+  source: string;
+};
```

Comment:
```suggestion
export type TitleData = {
  content: string;
  priority: number;
};

export type LinkData = {
  name: string;
  url: string;
};

export type LinkImage = {
  name: string;
  source: string;
};
```
Разделяй типы пустой строкой